### ISSUE TITLE
Remove Weighted Map Vote

### ISSUE THREAD

**Issue by MusgardTiger** [d3athrow/vgstation13#9640]
Issue is very straight forward. Whenever this kicks in majority of the playerbase is forced to play on a map they don't want to and some or most of them leave the server depending on the current pop. 

If it was low or low-mid pop this turns the whole thing into deadpop.

**Comment by carshalash** [comment]
Just remove taxi.

**Comment by TheFriggenSawbones** [comment]
Yeah the 1 vote taxi/deff wins are total bullshit and whoever made this change needs to say they're sorry

**Comment by Shadowmech88** [comment]
I think Nexis was originally the one that made map votes weighted.

**Comment by TheFriggenSawbones** [comment]
we demand an apology

**Comment by uazuaz** [comment]
weighted vote is fine to me, otherwise you will play just box everytime.
an easy fix could be this: in the map vote selection, don't show the map that you are currently playing (with the exception of box), so you will not play the same map twice in a row (again, with the exception of box, because is the most popular map).

**Comment by iwillkillmyself** [comment]
Gas the Taxi, fix Mini, map war now

**Comment by ArchSunder** [comment]
Taxi is the winner with 1 vote and a 2% chance!
What the fuck else needs to be said.

**Comment by MercuryDoll** [comment]
Easy fix: if a map has 5 times more votes than any other, it's not gonna be weighted. It's mostly the 1-vote bullshit that keeps happening, disqualify maps with less than 3 votes.

**Comment by uazuaz** [comment]
and again, you will only play box.
by the way rejecting the 1-vote maps for weighted vote could be ok.

**Comment by MusgardTiger** [comment]
If 30 people wants to play box and 5 people wants to play some other map, 5 people can suck it up.

This is not a kindergarten where you have to cater for every single person because they want to have something special, forcing majority to play on a map they don't want to is simply bad.

**Comment by SangriaMonkey** [comment]
The maps are just bad. Taxi might be funny to watch but if you want to actually do anything other than dick around it's a fucking nightmare to manage. We don't have enough players to support something as dense as Deff either. It should really just alternate between box and meta every round until someone makes a new map that isn't gimmicky to a fault.

**Comment by ihadtoregisterforthis** [comment]
Remove weighted vote if and only if we go back to random map.

**Comment by ghost** [comment]
ITT: People who don't understand statistics

**Comment by SerWind** [comment]
I don't think understanding statistics has anything to do with it. More like people don't understand why they are being forced to play maps they don't want to for a minority, and simply writing it off as they are all idiots is kind of shitty.

**Comment by MusgardTiger** [comment]
Thumbs downed for being a memeing fuckhead @DrCelt 

Thumbs up'd for understanding the core of this issue. @Serwind

Edit: Literally just happened 
![wowgreatfeature](https://cloud.githubusercontent.com/assets/10194313/14784655/5938cb88-0aff-11e6-8814-99e48898ed0e.png)

**Comment by Senaat1** [comment]
Weighted map voting is fine. I don't really like any other station but Box, but that doesn't mean other players don't want to play something else every now and then.

**Comment by MusgardTiger** [comment]
Senaat1 that is your opinion and when I read it it makes sense but you are not a regular player and your opinion is unfortunately too naive. There is a reason why we are playing box a whole lot than any of these maps because it is better and not a desolate maint wasteland like deff or just shitty like taxi. I personally like meta a lot but I don't think it is right to force 10 or 50 players to play it when they don't want to.

**Comment by Senaat1** [comment]
> but you are not a regular player and your opinion is too unfortunately too naive.

Mustard, i have been playing this game for four years on a regular basis, how about you fuck off with that shit? Maybe go play on a different server if you're going to be a crybaby you newfag.

**Comment by MusgardTiger** [comment]
I meant you aren't playing on /vg/ station regularly anymore. I didn't mean you started playing yesterday.

**Comment by Senaat1** [comment]
Even then, i still play a round or two a day. And if 49 people don't want meta and one person does, guess what, they'll get to play meta roughly one of every fifty rounds. If they can't handle that, that's their problem.

**Comment by MusgardTiger** [comment]
I simply don't agree with that and my reasoning for it is in OP. This effects low and low-mid pop more than it does mid to high pop which I usually see you in and it is a legitimate problem for the server. 

I'd be personally okay with this if we had better maps in rotation but both taxi and deff demands a lot effort from players for just to do their jobs. On the other hand box and meta get voted consistently, meta votes are nowhere close to box votes but still it isn't like one meme vote for taxi and three votes for deff (if it gets voted at all).

Let me put this down as well these are my personal observations and I didn't go through numbers a whole lot.

Edit: I'd even be okay if we removed weighted vote when player number is lower than 25, which is around when weighted vote causes a real damage.

**Comment by PJB3005** [comment]
# I MADE WEIGHTED VOTING

Fuck y'all, no regrets.

**Comment by CptWad** [comment]
in the end it boils down to a few things

\>People who cant stand change and hate everything that is not box
\>People who cant build the SME and cause deff to suffer
\>People who don't understand how to power taxi and cause it to suffer

The first thing is unfixable but the other two is just engineers who are not as robust. Classically engineers have always been bad at their jobs and usually can only ever set up the primary power source, in our case the singlo, given that its not set up mostly for them they usually cant set up power then bumble around engineering which manages to get a shuttle call and everyone complaining that deff is shit
which is why whenever I get to play on deff I will set up the SME because I know how to
Additionally to the edges of taxi and deff, is that people deliberately make the rounds worse to make deff and taxi end sooner because they dislike the map. Over and over again people have speed bombed and plasmaflooded and god knows what else to make the map shittier just because they think they are doing a good thing by ending the map as such
there are hot opinions on deff and taxi but at least some of it is not ones own opinion but one formed of others meming 

So I say we add more maps and see what happens
https://github.com/CptWad/vgstation13/issues/1

**Comment by SerWind** [comment]
Maybe people just don't like taxi because it's an interesting idea thats gimmick wears thin really quickly? Not because of strictly incompetent engineers or memes or fear of change. Yeah sure I imagine there are people like that. I enjoy deff from time to time. I like meta a lot. Box is great. 

Taxi is taxi.

**Comment by SerWind** [comment]
Also weighted voting was an interesting idea and I really dislike the fact we can't even discuss alternatives or outright removing it. Like weighted voting is infallable or the most perfect system this server has ever designed. Sorry PJB.

**Comment by SysadminJeroen** [comment]
This is what it was like before weighted votes.

![-vg-station map voting pre-weighted votes](https://cloud.githubusercontent.com/assets/2915526/14789410/a3b36756-0b0c-11e6-9586-9e1edbf0fa0b.png)

**Comment by Shadowmech88** [comment]
And yet even now with weighted votes, the majority _still_ almost always wants to play Box.

**Comment by MusgardTiger** [comment]
![a3b36756-0b0c-11e6-9586-9e1edbf0fa0b](https://cloud.githubusercontent.com/assets/10194313/14789868/4b38a798-0b17-11e6-942b-8285178552de.png)

I hope this helps t b h

**Comment by MercuryDoll** [comment]
As someone who really enjoys exploring new maps and hence don't consider myself a "box autist", i have to agree with the most common arguments. Meta is, after you learn where is what, similar to Box: a fluent map that is playable low to high pop. I didn't play much in most departments so i can't say how efficient they are in their workplace, but overall the map was decently to navigate.
Deff is fun with the massive abandoned station parts in the maintenance tunnels, but incredibly empty on lowpop and even on medpop you hardly see anyone. It has its issues but if you want it to be played for people to voice opinions and post issues for improvement, do that during med-hi pop.

Taxi is super gimmicky. It's a map you might want to play now and then, probably because admins want to bus it. But you don't want it for a common map. Using the taxi-shuttles was interesting the first two, maybe three times but after that it becomes just tedious and hindering, like a boulder you have to climb every time you want to go somewhere. I liked that the mines were right next to the station, but i can't imagine the hassle for medbay

tl;dr Deff and Meta are fine and improvable, Taxi is too gimmicky for regular gameplay, **especially lowpop** which is the biggest issue here.

**Comment by PJB3005** [comment]
> and I really dislike the fact we can't even discuss alternatives or outright removing it.

What?

**Comment by xpcybic** [comment]
Okay like
I get disliking taxi because its gimmick can be difficult to work with
But why are people posting their problems with maps in here instead of making issue reports
I've had to watch threads and ooc to get feedback for taxi and I'm sure deff gets about the same treatment. The whole reason weighted voting got added was to help underplayed maps get more feedback so they could be improved. Now that we've had time to improve them (and considering people abuse voting) I'd be fine for bumping up the threshold but ffs don't make us hunt for feedback. I'm hardly ever on at low enough pop to play taxi myself so I can't gauge how well it works in practice. Removing weighted vote even now will ensure not-box never sees the light of day again

On another note @iwillkillmyself ministation does still compile and it sounds like there's some support for adding it

**Comment by MusgardTiger** [comment]
@xpcybic  Biggest problem with Taxi is having to take cabs to go to other departments which is tedious. Taxi is fun in idea and first few rounds are fun because you are taking taxis around having fun with the new feature but when you stop enjoying the taking taxi to other ministations feature it becomes more of an obstacle rather than being an entertaining feature.

Which is fine when forced by admins every now and then for various bus ideas but not when it is in regular rotation.

**Comment by uazuaz** [comment]
replace cabs on taxi with teleporters.

**Comment by CptWad** [comment]
bus station when

**Comment by MercuryDoll** [comment]
> teleporters

This sounds stupid enough to work.

**Comment by SerWind** [comment]
Any time this topic is brought up it seems
1)the people against weighted mention points why it blows 
2)the people for it say fuck off, meme, insult, or just say it should stay and that's that. 

For the most part.

**Comment by uazuaz** [comment]
> replace cabs on taxi with teleporters.

expanding the concept:

-you don't need to wait for the cab to arrive
-you dont need to click on the console and wait for the cab to depart
-you'll be teleported to destination instantly just walking on it
-you can place multiple teleporters on each mini-station

**Comment by CptWad** [comment]
But Serwind
Only two people against you and mustard meme'd to any degree, one of who was just taking the credit for creating the system
xpcybic, MercuryDoll, Jeroen52, Senaat1, uazuaz and me have all put points for weighted votes
You are speaking from nowhere, we have clearly made the point that without the vote as it is theres no way other maps would get played because of all the votes for box.
If meta or deff wins by a small margin every once and a while then its not that huge of an issue. Maps that you dislike get played maybe once or twice a day. 
most of the time its box anyways, its a self correcting system, if there happens to be a few deff rounds in a row less people usually vote for it and deff wont end up the winner.

**Comment by Senaat1** [comment]
Wind, it's more like this:

People against it's argument boils down to "more people want this so we should do this 100% of the time"

People for it's arguments boil down to "let people play meta/deff every once in a while, box still wins the vast majority of the time anyway. Don't act like the anti-weighted camp is the reasonable intellectual side while the pro-weighted camp is the memers and shitposters, Mustard's argument against my opinion was literally "well you're naive so????"

**Comment by MusgardTiger** [comment]
My argument for your post wasn't "lol ur naive" that's what I thought about your opinion. My argument against your opinion was everything in that post which was written after the point I said that your opinion was naive. Which was basically explaining why I thought your opinion was naive and in my opinion letting bad maps run for the sake of variety is pretty naive because it causes players to leave the server.

I think SerWind got heated up because of Celt's shitty statistics meme and Cpt Wads opinion about the subject (which was a well written post but it simplified the people who are against weighted voting as people who can't handle anything but box and can't set up SME in other maps which might be the case for few people but simplifying this whole thing like that is pretty fuccin bad.)

**Comment by SerWind** [comment]
Yeah you are both right, people have brought up points in defense of weighted IE box being played all the time. I really don't think just flat out removing it is the solution though, that's never been my position. My argument has been against tiny minorities dictating the next map.

**Comment by Senaat1** [comment]
I'd suggest we just remove Taxi.

**Comment by TheFriggenSawbones** [comment]
nuke deff

**Comment by SerWind** [comment]
Or that.

**Comment by MusgardTiger** [comment]
Removing taxi and having it as a gimmick map and making sure deff wouldn't show up on the vote list if there aren't at least 40 players and along with that disabling weighted vote if we don't have at least 25 players online would minimize the problems we are having with this voting system.

Weighted vote is what enabling this meme vote forced shitty map situation and it is very arbitrary but now that Cpt_Wad is coming up with his own map I can see how disabling it completely would hinder the progress of a new /vg/ map. Which makes me relate to arguments of the other """"camp"""" but if weighted vote is to stay it should be improved and the idea I put down in this post would improve the situation a lot in my opinion.

**Comment by CptWad** [comment]
duny said hed be back during summer or something which would mean more deff updates
id probably finish donut by then too
We could also try to invent some other new way of map choosing
hell lets dig around other games and see how they decide map selection.
We could do something like splatoon map selection or otherwise. While DEMOCRACY is good and all the reason box is so preferred is because its broken in after lots of use. More playing deff would aid in establishing better relations with it, same with other maps. Before the voting everyone thought meta was terrible (at least my impressions of ooc) but after a while its grown to be "ok" in some peoples minds.

**Comment by Senaat1** [comment]
Probe has been improving deff, so we'll get deff updates soon.

**Comment by MercuryDoll** [comment]
> lets dig around other games and see how they decide map selection

Valve games do straight majority vote, on a tie it's RNG between the winners. So do most other games. Splatoon rolls maps completely random out of a rotating pool. We're currently the most close to MarioKart8 voting, where people can suggest a map and RNG picks one of them.

To recap, the major problem is 
1. Taxi being a gimmick map, and people don't want gimmicks to be repeated that often. 
2. Weighted voting having much more impact on lowpop and "grief voting" being a thing.
All we need to do is remove maps from lowpop voting that people refuse to play at all in lowpop.

**Comment by AutismSpider** [comment]
@Senaat1 yes i have done fuck all we now have a Rust and TEG combo that do nothing the TEG has been made useless and no one wants to unfuck how the Rust makes rads it work fine befor and now it does not

**Comment by ihadtoregisterforthis** [comment]
It's easier on other games where if you don't like the map or the rules you can just go to one of the other thousand servers, and find the one you like the most. Stuff like 24/7 nocrits 2fort is a possibility; stuff like 24/7 metaclub vgstation flavored only server is extremely hard.
Here we are all bundled together like sticks, and giving up the control over the map selection once in a while is needed for cohabitation. Giving the control by force, with weighed random vote, if you can't share playing time.

**Comment by mrpain666** [comment]
It needs to fucking go now.

**Comment by carshalash** [comment]
Hey look it happened again.
